subiquity/server/controllers/identity.py

```diff
@@ -62,6 +62,8 @@ class IdentityController(SubiquityController):
         'additionalProperties': False,
         }
 
+    interactive_for_variants = {'desktop', 'server'}
```

Comment:
I think desktop wants to not be on this list.

Reply:
yeah but not yet